examples/bootstrap.php

```diff
@@ -71,3 +73,11 @@ function print_token_usage(Metadata $metadata): void
     echo 'Remaining tokens: '.$tokenUsage->remainingTokens.\PHP_EOL;
     echo 'Utilized tokens: '.$tokenUsage->totalTokens.\PHP_EOL;
 }
+
+function print_vectors(ResultPromise $result): void
+{
+    assert([] !== $result->asVectors());
+    assert(array_key_exists(0, $result->asVectors()));
+
+    print_vectors($result);
```

Comment:
one replace too much here :)
```suggestion
    echo 'Dimensions: '.$result->asVectors()[0]->getDimensions().\PHP_EOL;
```